.github/workflows/ci.yml

```diff
@@ -23,26 +27,14 @@ jobs:
         - name: Test
           run: make test
     verify:
-        strategy:
-          matrix:
-            go-version: [1.15.x]
-            os: [ubuntu-latest]
-        runs-on: ${{ matrix.os }}
+        runs-on: ubuntu-latest
         steps:
-        - name: Install Go
-          uses: actions/setup-go@v2
-          with:
-            go-version: ${{ matrix.go-version }}
-        - name: Check out code
+        - name: Checkout code
           uses: actions/checkout@v2
-        - name: Verify fmt
-          run: make verify-fmt
-        - name: Verify generated code
-          run: make verify-generate
-        - name: Verify generated manifests
-          run: make verify-manifests
-        - name: Verify OLM bundle
-          run: make verify-bundle
```

Comment:
These checks need to stay - we have client/deepcopy functions that are auto-generated, and the YAML manifests for the bundle need to be kept in sync.

Reply:
I've pushed a new commit that rebased on latest main and restore those checks.